azext_iot/_params.py

```diff
@@ -819,6 +819,18 @@ def load_arguments(self, _):
             type=int,
             help="Maximum number of configurations to return. By default all configurations are returned.",
         )
+        context.argument(
+            "custom_metric_queries",
+            nargs="+",
+            options_list=["--custom-metric-queries"],
+            help="An altervative way of input style(narg key-value pair) to --metrics.",
```

Comment:
Spelling altervative -> alternative. We should say that this is intended to replace `--metrics` in the future.